code/modules/cargo/packs.dm

```diff
@@ -1597,14 +1658,24 @@
 /datum/supply_pack/service/noslipfloor
 	name = "High-traction Floor Tiles"
 	desc = "Make slipping a thing of the past with thirty industrial-grade anti-slip floortiles!"
-	cost = 2000
+	cost = 800
 	contains = list(/obj/item/stack/tile/noslip/thirty)
 	crate_name = "high-traction floor tiles crate"
 
+/datum/supply_pack/service/noslipfloor
+	name = "High-traction Floor Tiles"
+	desc = "Make an entire department not need to know the pain of slipping on a wet floor with 120 anti-slip floortiles!"
+	cost = 2000
+	contains = list(/obj/item/stack/tile/noslip/thirty,
+					/obj/item/stack/tile/noslip/thirty,
+					/obj/item/stack/tile/noslip/thirty,
+					/obj/item/stack/tile/noslip/thirty)
+	crate_name = "high-traction floor tiles crate"
+
```

Comment:
These 2 have the same datum. Also floor tiles rather than floortiles

Reply:
Done